samples/Basic_Delegates/app/application.cpp

```diff
@@ -9,19 +15,87 @@ public :
 		pinMode(ledPin, OUTPUT);
 	};
 	bool setTimer(int reqInterval) {
-		if (reqInterval <= 0) return false;
+		if (reqInterval <= 0) 
+		{
```

Comment:
Just to be consistent with the rest of the code, please, move the opening CB at the end of the previous line.